ecs-cli/modules/commands/flags/flags.go

```diff
@@ -151,6 +151,7 @@ const (
 	TaskDefinitionFile    = "task-def-file"
 	TaskDefinitionRemote  = "task-def-remote"
 	TaskDefinitionCompose = "task-def-compose"
+	AddOverrides          = "add-overrides"
```

Comment:
Nit: Do we want to call this `add-override` instead? Also I'm not convinced we shouldn't just call this `--override` and not have an abbreviation for the flag -- just so that it's explicit if the user is using them.

Reply:
I'd also maybe call the constant itself `ComposeOverrides`

Reply:
👍 Sounds good, yeah I wasn't happy with `add-overrides` either `--override` without a short name sounds better.

Done